gamification/labels.py

```diff
@@ -0,0 +1,51 @@
+POSITIVE_POINT_LABELS = (
+    'difficulty/newcomer',
+    'difficulty/low',
+    'difficulty/medium',
+    'difficulty/high',
+    'importance/low',
+    'importance/medium',
+    'importance/high',
+    'importance/critical',
+    'area/documentation',
+    'area/upstream',
+    'area/genericbears',
+    'area/lintbears',
+    'area/nativebears',
+    'type/bug',
+    'type/feature',
+    'bear/proposal',
+)
+
+NEGATIVE_POINT_LABELS = (
+    'status/invalid',
+    'status/duplicate',
+)
+
+
+def get_sorted_labels(labels):
```

Comment:
this *function* should not be in gamification/data , but the data above should be in gamification/data !